src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
Luckily the next deadline is in 2 weeks, so you have some time :smile: Not sure if we will use the DevHub feature if this enhancement is not included, as the gist already does the work and it is crucial to our TA assignment. If you want to add it in a different PR or now, I don't mind either way :stuck_out_tongue: